content/en/docs/Writing policies/policy-settings.md

```diff
@@ -12,7 +12,7 @@ A [policy](/docs/kyverno-policies/) contains one or more rules, and the followin
 
 * **validationFailureActionOverrides**: a ClusterPolicy attribute that specifies `validationFailureAction` Namespace-wise. It overrides `validationFailureAction` for the specified Namespaces.
 
-* **background**: controls if rules are applied to existing resources during a background scan. Defaults to "true".
+* **background**: controls scanning of existing resources to find potential violations and creating report of those violations during a background scan and is enabled by default.
```

Comment:
"controls scanning of existing resources to find potential violations and generating Policy Reports. See the documentation here (insert link). Default to "true""

Reply:
👍